content/tools/_index.md

```diff
@@ -2,14 +2,25 @@
 title: Tools
 ---
 
-We maintain a collection of tools, small and large, that help projects across the ecosystem:
+We maintain a collection of tools, small and large, that help projects across the ecosystem.
+All tools:
+
+- are released under the [3-Clause BSD License](https://opensource.org/license/bsd-3-clause/);
```

Comment:
If we migrate something, it may already have a non BSD-3 license. I don't think it's a show stopper though, or is it?

Reply:
If we don't want to have any issues, I would say so. They can re-license to comply with that. Or we can allow things like MIT. But it will be a real headache otherwise 😅

Reply:
I would slightly prefer to be overly restrictive to begin with (i.e., only accept 3-Clause BSD Licensed code). @bsipocz Are there any existing projects that you've been discussing bringing over that don't?

For now, I just want to get up something simple and cleanup the existing tools, bring in a handful of other tools, and then set up a slightly more formal process and guidelines. Of course, the simple thing could be:
```
are released under a permissive open source license (e.g., the [3-Clause BSD License](https://opensource.org/license/bsd-3-clause/));
```
or
```
are released under a permissive open source license (preferably the [3-Clause BSD License](https://opensource.org/license/bsd-3-clause/));
```

Reply:
I'd stick with BSD-3 until others are needed, and then re-evaluate. It should be fairly straightforward to re-license small utilities? MIT is fine, but others (like Apache) can be trickier to incorporate.

Reply:
> Are there any existing projects that you've been discussing bringing over that don't?

No. But e.g. some tutorial content may better be served by a CC license than a BSD, and we do have and will have those documents in the org. 
Also, if something is MIT, do we really need to go through the loops of relicensing or could just take it as is?

So it's really about BSD-3 for new things (or CC when it's content rather than code), but be a bit more flexible if it's migrated over.

Reply:
Good point about CC for docs 👍 We could allow that. For the MIT, BSD, maybe we should say: directly compatible. As Stéfan said, we can wait for the occasion to be there to deal with that.

Reply:
But this is really just a minor comment, stating that BSD-3 is what we do is better than the current not saying any preference.

Reply:
I would be more welcoming to those preferring MIT and said that the package needs to be either BSD or MIT. That removes a bit of potential fraction without causing any trouble. CC for documentation and other resources makes also sense but there I would be restrictive in which CC flavours we'd accept, because for example CC-BY-NC-ND ([Creative Commons Attribution-NonCommercial-NoDerivatives](http://creativecommons.org/licenses/by-nc-nd/4.0/)) cannot be considered open and permissive.

Reply:
Those two licenses are similar enough that I'd be happy for either.

Reply:
I updated to include both BSD-3 and MIT in 9f010377ac43b87d7b43e97d51f663b3a0418d4c. I'll open a separate PR to add a section somewhere about documentation. Let's continue the discussion there.

Reply:
See https://github.com/scientific-python/scientific-python.org/pull/452.

Reply:
What do you think about allowing Apache v2? That's now the most popular license, surpassing MIT: https://www.mend.io/blog/open-source-licenses-trends-and-predictions/.

Reply:
I think for that we need a proper legal advice. I am not sure how to mix Apache with MIT and BSD. Can NumFOCUS help?

Reply:
Let's table Apache v2 for now and revisit it in a follow-up PR.

Reply:
Apache is GPL2 incompatible.

Reply:
> What do you think about allowing Apache v2?
> ...
> Apache is GPL2 incompatible.
> ...
> Let's table Apache v2 for now and revisit it in a follow-up PR.

I'm not very informed when it comes to the subtleties of license edge cases, so whatever this follow up PR is would be great to learn from in general (several tools I maintain are Appache v2).

In terms of phrasing, 

```markdown
- are released under a permissive open source license (preferably the [3-Clause BSD License](https://opensource.org/license/bsd-3-clause/) or or [MIT License](https://opensource.org/license/mit/));
```

would probably get my vote as it makes it clear what the stance is, yet doesn't dictate anything. That being said I think I'm happy to defer my opinion on licenses to everyone else who is more informed here.